app/admin/restaurant.rb

```diff
@@ -35,4 +35,11 @@
     end
     f.actions
   end
+  show do
+    attributes_table do
+      row :image do |ad|
+        image_tag ad.image.url
```

Comment:
Is the image the only thing you are showing? Please generate a screenshot using `Then show me an image of the page` and attach it to the PR.